content/docs/intro/concepts/programming-model.md

```diff
@@ -1005,7 +1013,7 @@ var vpc = new MyVpcComponent("vpc", new ResourceOptions
 });
 ```
 
-Transformations can also be applied in bulk to many resources in a stack by using the `pulumi.runtime.registerStackTransformation` function.
+Transformations can also be applied in bulk to many resources in a stack by using the `pulumi.runtime.registerStackTransformation` or `ctx.RegisterStackTransformation` function.
```

Comment:
This might be confusing the users. We should either have a language choose here, or just refer to the name of the method without the namespacing.